docs/signing_release_candidate.md

```diff
@@ -0,0 +1,66 @@
+# Signing a LND release candidate
+
+When a new version of LND is released, binaries for the lnd and lncli programs
+are provided for various platforms and CPU architectures. The hashes of all
+these binaries are written into a file known as the "Manifest". This Manifest is
+then signed by at least five trusted LND contributors (see `[verify-install.sh](https://github.com/lightningnetwork/lnd/blob/master/scripts/verify-install.sh)`
+script for more details), ensuring that users can trust the binaries they 
+download, knowing they haven't been tampered with or backdoored.
+
+To verify a release binary, users have two options:
+
+* Manual Verification: Users can manually download the signature files and 
+Manifest from GitHub LND release page, then verify the PGP signatures and 
+hashes.
+
+* Automated Verification: The LND repository provides a script, 
+`[verify-install.sh](https://github.com/lightningnetwork/lnd/blob/master/scripts/verify-install.sh)`, that automates the verification process. This script uses a
+set of trusted developer keys (located in the repo under `[scripts/keys/](https://github.com/lightningnetwork/lnd/tree/master/scripts/keys)`) and
+downloads the necessary data from the GitHub server to verify the integrity of
+the local LND/LNCLI binaries.
+
+This process helps ensure that the release binaries are legitimate and secure.
```

Comment:
Again, they just prove the code wasn't altered by the GitHub build. "Secure" is asserted by reviewing the code itself.